.github/workflows/bindings_c.yml

```diff
@@ -0,0 +1,65 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Bindings C CI
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - '*'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "bindings/c/**"
+      - ".github/workflows/bindings_c.yml"
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup Cpp (C++ / C)
+        run: |
+          wget "https://github.com/aminya/setup-cpp/releases/download/v0.26.2/setup-cpp-x64-linux"
+          chmod +x ./setup-cpp-x64-linux
+          sudo ./setup-cpp-x64-linux --compiler llvm --ccache true
+          source ~/.cpprc
```

Comment:
Why we need this?

Reply:
BTW, it's not allowed in an ASF project to running external action like this.

Reply:
> Why we need this?

We need C++ compiler with c++14 in the machine, I can try removing this to check if the native compiler works fine.

Reply:
> 

> BTW, it's not allowed in an ASF project to running external action like this.

Sure, I will try remove it first. If it is not working, I will use apt to install all these.

Reply:
I remove it, and the CI worked out great 🔥